### PR TITLE
G Suite: Update Pending ToS notices and dialog for Google Workspace

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request updates the dialog displayed when a user has not accepted the terms of service of a G Suite user yet in order to also support Google Workspace. It also refines the notices and the copy of the dialog with simpler and more accurate language:

Before | After
------ | -----
B1 | A1

Before | After
------ | -----
B2| A2

Before | After
------ | -----
B3 | A3

Finally, this pull request fixes an error message, a file permission as well as a placeholder.

#### Testing instructions

1. Run `git checkout update/pending-tos-notice-dialog-for-google-workspace` and start your server, or open a [live branch](https://calypso.live/?branch=update/pending-tos-notice-dialog-for-google-workspace)
2. Log into a WordPress.com account with a Google Workspace account with inactivated users
3. Open the [`Email` page](http://calypso.localhost:3000/email)
4. Assert that you see a notice inviting you to activate the Google Workspace users
5. Click the `Finish Setup` button
6. Assert that G Suite is no longer mentioned in the dialog

> Note you will have to apply D56184-code to be able to reset a password of a Google Workspace user.

### CODE REVIEW

File: client/lib/gsuite/new-users.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ const validEmailCharacterField = ( { value, error }: GSuiteNewUserField ): GSuit
 	error:
 		! error && ! /^[0-9a-z_'-](\.?[0-9a-z_'-])*$/i.test( value )
 			? translate(
-					'Only number, letters, dashes, underscores, apostrophes and periods are allowed.'
+					'Only numbers, letters, dashes, underscores, apostrophes and periods are allowed.'
```
Comment (a8ci18n) [inline client/lib/gsuite/new-users.ts:102]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice-dialog.jsx [hunk 2]
```diff
@@ -165,11 +166,11 @@ function PendingGSuiteTosNoticeDialog( props ) {
 					<Button
 						href={ getLoginUrlWithTOSRedirect( props.user, props.domainName ) }
 						onClick={ onResetPasswordLogInClick }
-						primary={ isCopied }
+						primary
```
Comment (stephanethomas) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice-dialog.jsx:169]: We can make this button always primary as we shouldn't assume that the user will use the `Copy` button.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx [hunk 3]
```diff
@@ -136,14 +137,13 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 			<>
 				<p>
 					{ translate(
-						'Your mailbox {{strong}}%(emails)s{{/strong}} is almost ready! Complete the setup to activate your new email address.',
-						'Your mailboxes {{strong}}%(emails)s{{/strong}} are almost ready! Complete the setup to activate your new email addresses.',
+						'Your new user {{strong}}%(emails)s{{/strong}} is ready! Complete the setup in order to activate it.',
+						'Your new users {{strong}}%(emails)s{{/strong}} are ready! Complete the setup in order to activate them.',
```
Comment (stephanethomas) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: We say in the transactional emails that the _users are ready_ so I've updated the copy to be more consistent here.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx [hunk 4]
```diff
@@ -169,11 +169,13 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 				showDismiss={ false }
 				key="pending-gapps-tos-acceptance-domain"
 				text={ translate(
-					'%(exclamation)s To activate your email {{strong}}%(emails)s{{/strong}}, click "Finish Setup".',
-					'%(exclamation)s To activate your emails {{strong}}%(emails)s{{/strong}}, click "Finish Setup".',
+					'%(exclamation)s Finish the setup of {{strong}}%(emails)s{{/strong}} in order to activate it.',
+					'%(exclamation)s Finish the setup of {{strong}}%(emails)s{{/strong}} in order to activate them.',
```
Comment (stephanethomas) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: There is no need to invite the user to click the `Finish Setup` as this should be pretty obvious.

File: client/my-sites/email/email-management/gsuite-users-card/placeholder.jsx [hunk 5]
```diff
@@ -18,7 +17,8 @@ import './style.scss';
 
 const Placeholder = () => (
 	<div className="gsuite-users-card__container is-placeholder">
-		<SectionHeader label={ 'G Suite Users' } />
+		<SectionHeader />
+
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/gsuite-users-card/placeholder.jsx:21]: We don't display any text in the placeholder so we can remove that label.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx [hunk 6]
```diff
@@ -70,9 +70,10 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 
 	getExclamation( severity ) {
 		const { translate } = this.props;
+
 		const translationOptions = {
-			context: 'Beginning of Gapps pending account notice',
-			comment: "Used as an exclamation in Gapps' pending account notice",
+			context: 'Beginning of G Suite pending account notice',
```
Comment (daledupreez) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: Should we not change this to "Google" or "Google Workspace"? "G Suite" has a pretty limited lifespan at this stage!

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx (hunk 3 above)
Comment (daledupreez) [reply client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: I think the second sentence could be simplified as:
> Complete the setup to activate it/them.

The "in order" here makes the sentence feel complicated to me.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx [hunk 7]
```diff
@@ -136,14 +137,13 @@ class PendingGSuiteTosNotice extends React.PureComponent {
...
-						'Your mailboxes {{strong}}%(emails)s{{/strong}} are almost ready! Complete the setup to activate your new email addresses.',
+						'Your new user {{strong}}%(emails)s{{/strong}} is ready! Complete the setup in order to activate it.',
+						'Your new users {{strong}}%(emails)s{{/strong}} are ready! Complete the setup in order to activate them.',
 						{
 							count: users.length,
 							args: { emails },
 							comment:
-								'%(emails)s will be a list of email addresses separated by a comma, ' +
-								'e.g. test@example.com, test2@example.com',
+								"%(emails)s is a list of email addresses separated by a comma (e.g. 'one@example.com, two@example.com')",
```
Comment (daledupreez) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: Might this be clearer if we state it as:
> %(emails)s is a list of email addresses separated by commas (e.g. 'one@example.com, two@example.com')

Not sure if that actually has any material benefit, so feel free to skip this to keep the existing comment closer to the existing one.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx (hunk 4 above)
Comment (daledupreez) [reply client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: As above, maybe remove "in order" to keep things simple.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx [hunk 8]
```diff
@@ -202,7 +204,7 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 				key="pending-gapps-tos-acceptance-domains"
 			>
 				{ translate(
-					'%(exclamation)s To activate your new email addresses, click "Finish Setup".',
+					'%(exclamation)s Finish the setup of your new users in order to activate them:',
```
Comment (daledupreez) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: Same again RE removing "in order" for simplicity.

**Review by daledupreez — APPROVED** [review]
Other than some minor tweaks to the wording, this looks good to me. :shipit:

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx (hunk 6 above)
Comment (stephanethomas) [reply client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: I wasn't sure about that one, I'll update it.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx (hunk 3 above)
Comment (stephanethomas) [reply client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx]: Thanks for your feedback, I'll update all those strings as you suggest.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5512689

Thank you @stephanethomas for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5512689).

**Pull request merged**